src/trips/dto/create-trip.dto.ts

```diff
@@ -1,9 +1,51 @@
-import { IsNotEmpty } from 'class-validator';
+import { IsArray, IsBoolean, IsNotEmpty, IsNumber, IsOptional, isString, IsString } from 'class-validator';
+
+import { Currency } from '../../enums/currency.enum';
+import { Level } from '../../enums/level.enum';
 
 export class CreateTripDto {
   @IsNotEmpty()
+  @IsString()
   public name: string;
 
-  @IsNotEmpty()
-  public user: string;
+  @IsString()
+  public description: string;
+
+  @IsString()
+  public location: string;
+
+  @IsNumber()
+  public price: number;
+
+  @IsString()
+  public level: Level;
+
+  @IsString()
+  public currency: Currency;
+
+  @IsString()
+  public material: string;
+
+  @IsString()
+  public including: string;
+
+  @IsString()
+  public amenities: string;
+
+  @IsNumber()
+  public duration: number;
```

Comment:
add startTime also, we forgot to tell you about it

Reply:
done